### DISCUSSION TITLE
日本の翻訳コミュニティのロゴデザインの投票

### DISCUSSION THREAD

**Discussion by gurezo** [mozilla-japan/translation#762] [Polls]
### l10n 国旗入りバージョン x 2(白黒`赤`、白黒)
![スクリーンショット 2024-05-13 22 43 20](https://github.com/mozilla-japan/translation/assets/7631567/4487146c-dad1-4ef9-b484-ade2f2b08ac6)

### Japan Community バージョン x 2(白黒`赤`、白黒)
![スクリーンショット 2024-05-13 22 43 45](https://github.com/mozilla-japan/translation/assets/7631567/5d598e8a-98bc-4d4e-bb10-8ea90e9dd937)

### Japan localization バージョン x 2(白黒`赤`、白黒)
![スクリーンショット 2024-05-13 22 44 07](https://github.com/mozilla-japan/translation/assets/7631567/32505c98-102a-4385-a16e-f9c6749a42c2)

### Japan localization + 国旗 バージョン x 2(白黒`赤`、白黒)
![スクリーンショット 2024-05-18 15 24 57](https://github.com/mozilla-japan/translation/assets/7631567/93d98037-9dbc-4784-8040-6302e605abf7)

**Comment by hmatrjp** [comment]
投票呼びかけの文内、6つ → 8つ に訂正しようかと思ったのですが、修正すると投票がリセットさせるそうなのでやめました。

**Comment by marsf** [comment]
今更コメントするのもなんですが、外国では言語と国旗が直接結びつかないのでご注意を。
それから、"L10n Community" は昔から製品ローカライズのコミュニティを指して使われていますが、そこらへんの区別というか、どういう括りでしょうか？

つまり、以下のどの解釈が近いのか、という意味です。
"MDN Web Docs(, and) Mozilla l10n Community [Japan]",
"MDN Web Docs (with) Mozilla Japan Community",
"Mozilla MDN Web Docs Japan(,) localization" / "MDN Web Docs Mozilla Japan localization" ??,
"Mozilla MDN Web Docs Japan(,) localization [Japan]"

もし、単語を並べただけなのであれば、どれも第一印象で受け取り方が全く違うということを考慮して選んだほうが良いと思います。

**Reply by gurezo** [reply]
@marsf 
コメント有難うございます。

> 今更コメントするのもなんですが、外国では言語と国旗が直接結びつかないのでご注意を。 

それは知りませんでした。有難うございます。

> つまり、以下のどの解釈が近いのか、という意味です。

こちらについては日本の翻訳コミュニティーというイメージで、言葉を選びました。
提示して頂いた言葉からだと下記のキーワードがイメージに近いと思いました。

- "MDN Web Docs(, and) Mozilla l10n Community [Japan]",
- "Mozilla MDN Web Docs Japan(,) localization [Japan]"

ただ、赤枠のスペースを有効利用するデザイン上の理由で、結果的には単語を並べた状態になりました。
大急ぎで作成したので、相手にどのように伝わるかまでは考えられていませんでした。

<img width="526" alt="スクリーンショット 2024-05-19 13 47 45" src="https://github.com/mozilla-japan/translation/assets/7631567/61424c25-bb23-46a5-9e98-328a232d75f8">

https://github.com/mozilla-japan/translation/discussions/756 で、お話していますがキッカケは、OSC などのイベント出展で
来場者に興味を持って貰い易くする為の備品として提起しました。

@marsf のアドバイスとデザイン上のスペースや自分のイメージをを考慮すると下記の言葉が一番近いという認識を持ちました。

1. `Japan localization`
2. `localization [Japan]` 

@marsf 如何でしょうか？
問題なけれなけば、現時点で一番投票数がある Japan localization + 国旗 バージョンの上記 1 or 2 の文言を修正します。
※ 1 の場合なら、文言変更は不要でした

**Reply by himorin** [reply]
目に止まったので便乗で。
国旗が言語を表さないというのはその通り（国際化デザインマニュアルでもよくやる危険な間違いとして常にリストされてますね）ですが、Japan localizationもなんか違和感があります。l10nにつけるならJapaneseではないでしょうか。jaかjpか問題と同じくで。

**Reply by gurezo** [reply]
@himorin cc @marsf 
コメント有難うございます。

そうすると正しくは、 `Japanese Localization` になりますか？（もちろん国旗`無し`

> 国際化デザインマニュアルでもよくやる危険な間違いとして常にリストされてますね

勉強になります。有難うございます。

**Reply by gurezo** [reply]
@marsf cc @hmatrjp @potappo 

> それから、"L10n Community" は昔から製品ローカライズのコミュニティを指して使われていますが、そこらへんの区別というか、どういう括りでしょうか？

改めてよく読んでみて気付きました。
気にされているのは、この点なんですね。


MDN Web Docs とは別に製品のローカライズにもコミュニティがあるという認識がありませんでした。
気付かせて頂いて有難うございます。


※ L10n Community = MDN Web Docs というイメージでした。
※ kuma yari しか翻訳していなかった事が理由です。


自分が今回提案したデザインは、MDN Web Docs `専門`の翻訳活動のコミュニティというイメージです。

上記のイメージで言えば、下記の表現が正しいのでしょうか？
Japanese Localization  for MDN Web Docs 

如何でしょうか？
テーブルクロスのデザインには、上記のコメントの `Japanese Localization` が空きスペースに入るイメージです。

**Reply by marsf** [reply]
> 自分が今回提案したデザインは、MDN Web Docs 専門の翻訳活動のコミュニティというイメージです。

MDN の活動のみを意識したロゴであれば、候補のうち、
- Japanese Localization (日本語ローカライズ) 
- (Mozilla) Japan Community (日本コミュニティ) 

のどちらかですね。

要するに、@himorin さんの ``jaかjpか問題と同じくで。`` が意味するところの `ja` (Japanese; 日本語) と `jp` (Japan; 日本国) のどちらを表したいかということになります。

**Reply by hmatrjp** [reply]
@marsf さんと @himorin さんの指摘を受けて、改めて考えてみると、
私個人の感覚では Mar さんの並べてくれた中では、 "MDN Web Docs (with) Mozilla Japan Community", が一番近いです。

スペースが許すなら、（左枠に Mozilla があるので） Japanese Translation Community
とするのが一番かなと思うのですが、文字数が多すぎる？？

国旗ありだと、（もし海外でブースを構えるときに使うことがあれば、）「ああ、日本から来ているのか」とパッと見でわかりやすいかもと思います。
逆に、国内限定で使うなら、Japan も国旗も取り払って、翻訳コミュニティー のほうが収まりがいいですね。

**Reply by gurezo** [reply]
@marsf @hmatrjp 

お二人共アドバイス有難うございます。
使う単語や翻訳コミュニティの範囲も明瞭になったと思いました。

今回明確になったコミュニティの定義や下記４パターンの文言使ったデザイン投票を再度開きたいと思います。

- Japanese Localization (日本語ローカライズ)
- (Mozilla) Japan Community (日本コミュニティ)
- Japanese Translation Community
- Translation Community

予め参考程度にデザイン上の文字数感を掴む為にいくつか訂正したパターンを乗せたいます。

## Japan Localization
![MDN-Web-Docs-Jp-Part 2](https://github.com/mozilla-japan/translation/assets/7631567/45b6ac1c-ec94-4c66-9476-e7d55f2c70ef)

## Japan Community
![MDN-Web-Docs-Jp-Part 8](https://github.com/mozilla-japan/translation/assets/7631567/fb83b3c5-d21c-415b-8a7d-709b562bc0e7)

## Japanese Localization
![MDN-Web-Docs-Jp-Part 9](https://github.com/mozilla-japan/translation/assets/7631567/9b536186-13a6-4660-979a-045b0a4a69c9)

## Japanese Translation Community
![MDN-Web-Docs-Jp-Part 4](https://github.com/mozilla-japan/translation/assets/7631567/6a680c17-8ede-4506-9102-c97346536406)

## Translation Community
![MDN-Web-Docs-Jp-Part 7](https://github.com/mozilla-japan/translation/assets/7631567/7feb362c-9fe7-4939-aec5-f9fcf8ceb2d0)

## 白文字パターン
![MDN-Web-Docs-Jp-Part 3](https://github.com/mozilla-japan/translation/assets/7631567/38eb01c2-9e8c-44ed-8a36-3fb40e1e962c)

**Reply by gurezo** [reply]
明日05/20 まで待って特に異論がなければ、このスレッドを閉じようと思います。
そして別スレッドで改めて、デザイン投票を募りたいと思います！

コメントで、アドバイスをして下さった皆様、有難うございます！

**Reply by potappo** [reply]
@gurezo 
Japan か Japanese かでこだわるなら、
(Mozilla) Japan Community (日本コミュニティ) よりは (Mozilla) Japanese Community (日本コミュニティ) の方が良いと思います。
Slack 、GitHub、YouTube は Mozilla Japanese Community なので。
検索した限りでは、mozilla.jp と Facebook での表記は  Mozilla Japan Community になっているようで、必ずしも統一されているわけではないですが、参考まで。

**Reply by gurezo** [reply]
@potappo 
アドバイス有難うございます！
デザインは、`Japanese` で作成したいと思います！

**Reply by gurezo** [reply]
一旦こちらのスレッドは閉じます。
別途デザインし直したスレッドを開きます。
